npm/iptm/iptm.go

```diff
@@ -127,13 +128,23 @@ func (iptMgr *IptablesManager) UninitNpmChains() error {
 		return err
 	}
 
-	// For backward compatibility, we should be cleaning older chains.
-	// TODO(jungukcho): need to check K8s or NPM version and do it selectively
-	// to avoid unnecessary call.
+	// Clean old NPM chains. This is forward compatible with NPM v2.
 	allAzureChains := append(IptablesAzureChainList,
 		util.IptablesAzureTargetSetsChain,
 		util.IptablesAzureIngressWrongDropsChain,
 	)
+	currentAzureChains, err := linuxutil.AllCurrentAzureChains(iptMgr.exec, defaultlockWaitTimeInSeconds)
+	if err != nil {
+		metrics.SendErrorLogAndMetric(util.IptmID, "Error: failed to get all current AZURE-NPM chains")
```

Comment:
Is this just a benign error? If so can we add to the error message that states this isn't bad?

Reply:
Technically benign for v1, so I can say "Warning: ..."